internal/host/validator.go

```diff
@@ -737,51 +739,50 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 		return ValidationSuccess
 	}
 
+	v.log.Debugf("Maximum packet loss percentage allowed for host %s: %f", c.host.ID, *c.clusterHostRequirements.Total.PacketLossPercentage)
 	if len(c.host.Connectivity) == 0 {
 		return ValidationPending
 	}
-	hosts, err := v.validatePacketLossForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
-	if err != nil || len(hosts) > 0 {
-		return ValidationFailure
-	}
-	return ValidationSuccess
+	status, _, _ := v.validatePacketLossForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts, c.cluster.OpenshiftVersion, c.cluster.EmailDomain)
+	return status
 }
 
-func (v *validator) validatePacketLossForRole(host *models.Host, clusterRoleReqs *models.ClusterHostRequirements, hosts []*models.Host) ([]string, error) {
-
-	var connectivityReport *models.ConnectivityReport
+func (v *validator) validatePacketLossForRole(host *models.Host, clusterRoleReqs *models.ClusterHostRequirements, hosts []*models.Host, openshiftVersion, emailDomain string) (ValidationStatus, []string, error) {
 	connectivityReport, err := hostutil.UnmarshalConnectivityReport(host.Connectivity)
 	if err != nil {
-		return nil, fmt.Errorf("unable to unmarshall host connectivity for %s:%s", host.ID, err)
+		v.log.Warnf("unable to unmarshall host connectivity for %s:%s", host.ID, err)
+		return ValidationFailure, nil, err
 	}
 	failedHostIPs := map[string]struct{}{}
 	failedHostNames := []string{}
 	for _, r := range connectivityReport.RemoteHosts {
 		for _, l3 := range r.L3Connectivity {
+			hostname, role, err := GetHostnameAndRoleByIP(l3.RemoteIPAddress, hosts)
+			if err != nil {
+				v.log.Warnf("unable to determine host's role and hostname for IP %s: %s", l3.RemoteIPAddress, err)
+				return ValidationPending, nil, fmt.Errorf("unable to determine role and hostname for IP %s: %s", l3.RemoteIPAddress, err)
```

Comment:
same here - ValidationFailure instead of ValidationPending

Reply:
:+1: